views/layouts/default.lavender

```diff
@@ -83,4 +83,7 @@ body
                 a(href='/guide.html#langref') Language Reference
           li(class='#{active_class_about}')
             a(href='/about.html') About
+          li(class='#{active_class_download}')
```

Comment:
@SamuelDavis  This is good to know. I am trying to accomplish this a different way in my navbarsConquered branch.